src/Models/Model.php

```diff
@@ -338,14 +377,13 @@ public static function propertyKeys() : array {
 	 */
 	public function setAttribute( string $key, $value ) : ModelInterface {
 		$this->validatePropertyExists( $key );
-		$this->validatePropertyType( $key, $value );
```

Comment:
Scenarios this would fail that we ought to support:

```
class Fish extends Model {
	protected $properties = [
		'eggs' => 'int',
	];
...
```
And in use...
```
// This would fatal.
$fish = new Fish();
$fish->eggs = '10';
```

Reply:
While we don't want Fatal error, it does not feel like deleting is a good solution. I wonder what @borkweb thinks about this approach.